### PR TITLE
Fix Python 3.5 vs 3.7 issue

### LINKED ISSUE #65: devel server gives empty directory listing
I am prompted for a seed or "random" when I visit the devel server, but visiting that URL gives me an empty directory listing page.

### PR DESCRIPTION
**Pull request by int00h5525**
Closes #65

### CODE REVIEW

File: Dockerfile.moth-devel [hunk 1]
```diff
@@ -16,6 +16,5 @@ COPY devel /app/
 COPY example-puzzles /puzzles/
 COPY theme /theme/
 
-WORKDIR /moth/
```
Comment (nealey) [inline Dockerfile.moth-devel]: Is this required for the Python3.7 fix?

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ and comes with a JavaScript-based scoreboard to display team rankings.
 Running a Development Server
 ============================
 
-    docker run --rm -it -p 8080:8080 dirtbags/moth-devel
+    docker run --rm -it -p 8080:8080 -v /path/to/puzzles:/puzzles:ro dirtbags/moth-devel
```
Comment (nealey) [inline README.md]: My intent here was that the devel server should start up with the built-in puzzles unless you override it. So I feel like this should actually be a new recipe under "Using your own puzzle categories" in the Devel server section.

File: devel/devel-server.py [hunk 3]
```diff
@@ -43,7 +43,11 @@ class MothRequestHandler(http.server.SimpleHTTPRequestHandler):
     
     def __init__(self, request, client_address, server):
         self.directory = str(server.args["theme_dir"])
-        super().__init__(request, client_address, server)
+        try:
+            super().__init__(request, client_address, server, directory=server.args["theme_dir"])
+        except TypeError:
+            super().__init__(request, client_address, server)
+
```
Comment (nealey) [inline devel/devel-server.py]: I'm starting to wonder if we shouldn't just backport the entire http.server module from Python 3.7.

File: Dockerfile.moth-devel (hunk 1 above)
Comment (int00h5525) [reply Dockerfile.moth-devel]: No, but this path doesn't exist on the devel server, and nothing will be written here while it's running. It just confuses path-resolution in the HTTP server more

File: devel/devel-server.py (hunk 3 above)
Comment (pi-rho) [reply devel/devel-server.py]: only if you want to march along with upstream backporting all changes to it. :) Can you point to the core problem? Maybe we can put guards around it or "monkeypatch" the behavior out of it (a long-term Pythonic solution).

**Pull request merged**